.env.example

```diff
@@ -13,6 +13,8 @@ DO_ENDPOINT_URL=https://nyc3.digitaloceanspaces.com
 DO_SPACES_KEY=generateme
 DO_SPACES_SECRET=generateme
 
```

Comment:
Since the PR here adds a job that loads and updates data into a database, its nice to be able to test the full load, but not on prod.  This env variable attempts to do that; its an opt-in env variable that PROD does not use.  But as a dev, it can be enabled and the loader can be run from the tests themselves.

Alternatively, running the loads via Dagster interface and/or via the jupytern notebook also works.